rules/words/irregular_verbs.txt

```diff
@@ -0,0 +1,307 @@
+alighted
+alit
+arisen
+arose
+ate
+awaked
+awoke
+awoken
+bade
+beat
+beaten
+became
+become
+been
+began
+begot
+begotten
+begun
+bent
+bereaved
+bereft
+beseeched
+besought
+bet
+betted
+bid
+bidden
+bided
+bit
+bitten
+bled
+blessed
+blest
+blew
+blown
+bore
+born
+borne
+bought
+bound
+bred
+broadcast
+broadcasted
+broke
+broken
+brought
+built
+burned
+burnt
+burst
+bust
+busted
+came
+cast
+caught
+chose
+chosen
+clad
+cleaved
+cleft
+clothed
+clove
+cloven
+come
+cost
+could
+crept
+crew
+crowed
+cut
+dealt
+did
+done
+drank
+drawn
+dreamed
+dreamt
+drew
+driven
+drove
+drunk
+dug
+dwelled
+dwelt
+eaten
+fallen
+fed
+fell
+felt
+fled
+flew
+flown
+flung
+forbad
+forbade
+forbid
+forbidden
+forcasted
+forecast
+forecasted
+forgot
+forgotten
+forsaken
+forsook
+fought
+found
+froze
+frozen
+gave
+gelded
+gelt
+gilded
+gilt
+given
+gnawed
+gnawn
+gone
+got
+gotten
+grew
+ground
+grown
+had
+heard
+heaved
+held
+hewed
+hewn
+hid
+hidden
+hit
+hove
+hung
+hurt
+kept
+kneeled
+knelt
+knew
+knit
+knitted
+known
+laid
+lain
+lay
+leaned
+leant
+leaped
+leapt
+learned
+learnt
+led
+left
+lent
+let
+lighted
+lit
+lost
+made
+meant
+melted
+met
+might
+molten
+mowed
+mown
+paid
+penned
+pent
+pleaded
+pled
+proved
+proven
+quit
+quitted
+ran
+rang
+read
```

Comment:
Please remove `read`, `run` and other first-form verbs

Reply:
done 👍